src/SSHLibrary/abstractclient.py

```diff
@@ -988,6 +988,13 @@ def _get_put_file_destinations(self, sources, destination, path_separator):
                      for path in sources]
         return files, dir_path
 
+    def _format_destination_path(self, destination):
+        destination = destination.replace('\\', '/')
+        colon_mark = ':/'
+        if colon_mark in destination:
+                destination = '/' + destination.split(colon_mark)[-1]
```

Comment:
Do I get it right that the idea is to drop possible drive letter from Windows paths? If yes, this would be better:

```python
import ntpath

destination = ntpath.splitdrive('destination')[-1]
```

Notice that you need to explicitly use `ntpath.splitdrive()`, not `os.path.splitdrive()`, to make this work same way regardless the OS where the code is running.